shared/node.go

```diff
@@ -83,19 +83,19 @@ func (node *Node) Start() error {
 		return err
 	}
 
-	if err := node.GetBus().GetP2PModule().Start(); err != nil {
+	if err := node.GetBus().GetRPCModule().Start(); err != nil {
```

Comment:
Add a comment that the order of the Starts is important and should not be modified.